lightly/api/api_workflow_upload_dataset.py

```diff
@@ -35,6 +37,12 @@ def upload_dataset(self: ApiWorkflowClient, input: Union[str, LightlyDataset], m
             max_requests:
                 Maximum number of requests a single worker can do before he has
                 to wait for the others.
+            size:
```

Comment:
Using the term 'output' here is very misleading in my eyes, as no image is outputted. Instead, the images  are resized before they are uploaded, thus the variable is also more like size_image_to_upload.
You could also distinguish the three possible cases better e.g. with a newline:
- if size is a negative int: images are uploaded without resizing
- if size is a positive int: ....
- if size is a tuple of (height, width) ...